assets/sass/components/_manual-barcode-form.scss

```diff
@@ -0,0 +1,34 @@
+#manual-barcode-form {
+
+  .barcode-fields {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+
+    .govuk-form-group {
+      padding-right: 2rem;
+    }
+    .govuk-input {
+      text-align: center;
+    }
+  }
+
+  .example-barcode {
+    @media (min-width: 40.0625em) {
```

Comment:
Not sure if this is the best/correct value to use for a media query. I got the value from the rendered page in chrome dev tools from another element (that uses a GDS style), but I dont know if 40.0625em is what chrome is reporting it as after some computation/calculation; or if there is a sass variable I can use; or if its OK ?

There is no mention of using media queries on https://design-system.service.gov.uk/